### PR TITLE
photonvision updated to version 6

### PR DESCRIPTION
**Pull request by notcharlie2026**

### CODE REVIEW

File: src/main/cpp/configs/RobotConfigChassisBot_99979997.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // OR OTHER DEALINGS IN THE SOFTWARE.
 //====================================================================================================================================================
 // This file was automatically generated by the Team 302 code generator version 1.2.3.5
-// Generated on Thursday, February 15, 2024 11:44:55 PM
+// Generated on Monday, February 19, 2024 7:49:31 PM
```
Comment (witcpalek) [inline src/main/cpp/configs/RobotConfigChassisBot_99979997.cpp:17]: Something isn't looking right, why is this file gamed 99979997 again; its currently 9997  Did you not use the latest code generator to modify the code generator files?

**Review by witcpalek — CHANGES_REQUESTED** [review]

File: src/main/cpp/configs/RobotConfigChassisBot_9997.cpp [hunk 2]
```diff
@@ -64,24 +63,10 @@ void RobotConfigChassisBot_9997::DefineVisionSensors()
...
+	                               units::length::inch_t ( units::length::inch_t ( 4.25 ) ), //units::length::inch_t mountingYOffset, /// <I> y offset of cam from robot center (left relative to robot)
+	                               units::length::inch_t ( units::length::inch_t ( 18.5 ) ), //units::length::inch_t mountingZOffset, /// <I> z offset of cam from robot center (up relative to robot)
 	                               units::angle::degree_t ( units::angle::degree_t ( 0 ) ), //units::angle::degree_t pitch,          /// <I> - Pitch of camera
-	                               units::angle::degree_t ( units::angle::degree_t ( 180 ) ), //units::angle::degree_t yaw,            /// <I> - Yaw of camera
+	                               units::angle::degree_t ( units::angle::degree_t ( 0 ) ), //units::angle::degree_t yaw,            /// <I> - Yaw of camera
 	                               units::angle::degree_t ( units::angle::degree_t ( 0 ) ) ); //units::angle::degree_t roll,           /// <I> - Roll of camera
 	DragonVision::GetDragonVision()->AddCamera ( Placer, RobotElementNames::CAMERA_USAGE::PLACER );
-
```
Comment (reslip1) [inline src/main/cpp/configs/RobotConfigChassisBot_9997.cpp:73]: when you ran generator you didn't have the limelight-pintake, so it removed it for this robot

File: vendordeps/photonlib.json [hunk 3]
```diff
@@ -1,57 +1,57 @@
 {
```
Comment (reslip1) [inline vendordeps/photonlib.json:1]: why did this change?

**Comment by reslip1** [comment]
@nickfrisch , @Tan560 , we may want to redo this branch after the other merge from main

File: codeGenerator/configuration/RobotConfig/2024Crescendo.xml
Comment (nickfrisch) [inline codeGenerator/configuration/RobotConfig/2024Crescendo.xml:1]: Will be included in @Tan560's big merge

**Review by nickfrisch — CHANGES_REQUESTED** [review]
Remove all changes other than photonlib.json

**Comment by reslip1** [comment]
This was added to the main branch by @Tan560 and @muscatd75

**Pull request closed without merging**